modules/vocabularies/src/main/frontend/src/vocabularies-admin-page.jsx

```diff
@@ -17,10 +17,29 @@
 //  under the License.
 //
 
+import { Box, Grid, Typography } from "@material-ui/core";
 import React from "react";
 
 function VocabulariesAdminPage(props) {
-  return(<div></div>);
+  return (
+    <Grid container direction="column" justify="space-between">
+
+      <Grid item>
+        <Box pb={5} bgcolor="background.paper">
```

Comment:
Padding shouldn't be used for adding space between items. It is only for adding space between a border and the content inside that border.

```suggestion
        <Box bgcolor="background.paper">
```

Reply:
Why do you need to specify a background color?

```suggestion
        <Box>
```